tests/integration/loopback_test.py

```diff
@@ -75,9 +77,12 @@ def test_change_loopback_mtu_and_restore_back(self, loopback_cleanup):
                 }
             ]
         }
-        libnmstate.apply(desired_state)
+        apply_with_description(
+            "Set the mtu of the lookback to 12800", desired_state
+        )
         assertlib.assert_state_match(desired_state)
-        libnmstate.apply(
+        apply_with_description(
+            "Remove the lookback device",
```

Comment:
Gris' previous comment applies here as well, also there is a typo

Reply:
Thanks for pointing it out. I updated it.